tests/collections/collection.test.ts

```diff
@@ -277,6 +286,19 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
             assert.strictEqual(findResDocs.length, 1);
             assert.strictEqual(findResDocs[0]._id, idToCheck);
         });
+        it('should find & findOne ne document', async () => {
+            const insertDocResp1 = await collection.insertOne(createSampleDocWithMultiLevel());
+            const insertDocResp2 = await collection.insertOne(createSampleDoc2WithMultiLevel());
+            const idToCheck1 = insertDocResp1.insertedId;
+            const idToCheck2 = insertDocResp2.insertedId;
+            const filter = { '_id': { '$ne': idToCheck1 } };
```

Comment:
Would also be great to test with `filter = { '_id': { '$ne': idToCheck2 } };` just in case

Reply:
solved, thanks